### PR TITLE
Add standard style linter

### PR DESCRIPTION
**Pull request by agentlewis**
@RichardGarcia this pr adds standard styles to the project, some npm scripts for helping linting and formating as well as some git hooks.

this should provide a consistent rule check on the style of code we are writing.

### CODE REVIEW

**Comment by agentlewis** [comment]
with this you can now run `npm run lint` and it will tell you all the things that have been done but not wired up properly

File: ui/components/molecules/M003/stories.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { storiesOf } from '@storybook/react'
 
 import Nav from '.'
 
-storiesOf('Molecules|M003', module)
-.add('Default', () => (
+storiesOf('Molecules|M003', module).add('Default', () => (
   <Nav title={`'First item'`} />
 ))
```
Comment (RichardGarcia) [inline ui/components/molecules/M003/stories.js]: Ditto: I need to clean/update or delete this,  Added task in Asana.

File: package.json [hunk 2]
```diff
@@ -24,6 +18,31 @@
...
+  "lint-staged": {
+    "ui/**/**/**/**/**/*.js": [
+      "standard",
+      "git add"
+    ],
+    "domains/**/**/**/**/**/*.js": [
+      "standard",
+      "git add"
+    ]
+  },
```
[the remark this answers is not in the document]
Comment (agentlewis) [reply package.json]: we aren't using it yet - but its where redux stuff is going to happen

File: ui/components/atoms/A001-Button/stories.js [hunk 3]
```diff
@@ -6,9 +6,7 @@ import Button from '.'
```
[the remark this answers is not in the document]
Comment (agentlewis) [reply ui/components/atoms/A001-Button/stories.js]: this doesn't have anything to do with this pr?

File: ui/components/atoms/A004-AppBar/stories.js [hunk 4]
```diff
@@ -9,7 +9,11 @@ import Button from '.'
...
-  <AppBar position='static' color='default' appBarTitle='App bar title' />
-), { notes: { markdown: notes } })
+  .addDecorator(withNotes)
+  .add(
+    'App bar',
+    () => (
+      <AppBar position='static' color='default' appBarTitle='App bar title' />
+    ),
+    { notes: { markdown: notes } }
+  )
```
[the remark this answers is not in the document]
Comment (agentlewis) [reply ui/components/atoms/A004-AppBar/stories.js]: yes sometimes standard does some things im not a fan of. however i would rather code was consistent and readable over stylistically subjective

File: ui/components/molecules/M001/stories.js [hunk 5]
```diff
@@ -3,7 +3,6 @@ import { storiesOf } from '@storybook/react'
...
 import Nav from '.'
 
-storiesOf('Molecules|M001', module)
-  .add('Default', () => (
-    <Nav title={`'First item'`} />
-  ))
\ No newline at end of file
+storiesOf('Molecules|M001', module).add('Default', () => (
+  <Nav title={`'First item'`} />
+))
```
[the remark this answers is not in the document]
Comment (agentlewis) [reply ui/components/molecules/M001/stories.js]: again not to do with this pr

File: ui/components/molecules/M003/index.js [hunk 6]
```diff
@@ -9,9 +9,7 @@ import Button from '../../atoms/A001-Button'
 const styles = {}
 
 const M003 = props => {
-  return (
-    <Button title={props.title} color={'primary'} />
-  )
+  return <Button title={props.title} color={'primary'} />
 }
 
 M003.propTypes = createPropTypes(schema)
```
[the remark this answers is not in the document]
Comment (agentlewis) [reply ui/components/molecules/M003/index.js]: :+1:

File: ui/components/molecules/M003/stories.js (hunk 1 above)
Comment (agentlewis) [reply ui/components/molecules/M003/stories.js]: :+1:

**Pull request merged**